helmfile/values/thanos/objectstorage-secret.yaml.gotmpl

```diff
@@ -26,17 +26,13 @@ objstoreConfig: |-
   config:
     auth_version: {{ $swift.authVersion }}
     auth_url: {{ $swift.authUrl }}
-    {{- if getOrNil "region" $swift }}
-    region_name: {{ $swift.region }}
```

Comment:
Is region not needed any more? Or has it never been?

Reply:
It seems to work without, but no I did not intend to remove it (i think :P).